fe/fe-core/src/main/java/org/apache/doris/statistics/AnalysisHelper.java

```diff
@@ -0,0 +1,47 @@
+// Licensed to the Apache Software Foundation (ASF) under one
+// or more contributor license agreements.  See the NOTICE file
+// distributed with this work for additional information
+// regarding copyright ownership.  The ASF licenses this file
+// to you under the Apache License, Version 2.0 (the
+// "License"); you may not use this file except in compliance
+// with the License.  You may obtain a copy of the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+package org.apache.doris.statistics;
+
+import org.apache.doris.common.Config;
+import org.apache.doris.common.ThreadPoolManager;
+import org.apache.doris.common.ThreadPoolManager.BlockedPolicy;
+import org.apache.doris.common.util.MasterDaemon;
+
+import java.util.concurrent.Future;
+import java.util.concurrent.LinkedBlockingQueue;
+import java.util.concurrent.ThreadPoolExecutor;
+import java.util.concurrent.TimeUnit;
+
+
+/**
+ * Used to help collect statistics.
+ * TODO add other method
+ */
+public class AnalysisHelper extends MasterDaemon {
+
+    private final ThreadPoolExecutor executors = ThreadPoolManager.newDaemonThreadPool(
+            Config.statistics_helper_running_task_num,
+            Config.statistics_helper_running_task_num, 0,
+            TimeUnit.DAYS, new LinkedBlockingQueue<>(),
+            new BlockedPolicy("AnalysisHelper Executor", Integer.MAX_VALUE),
+            "AnalysisHelper Executor", true);
+
+    public Future<?> asyncExecute(Runnable runnable) {
+        return executors.submit(runnable);
+    }
+}
```

Comment:
Seems this class is only partially implemented....it even didn't overwrite runAfterCatalogReady, what's the meaning of commit this unrelated code in this PR?